packages/@ember/-internals/environment/lib/env.ts

```diff
@@ -36,8 +36,6 @@ export const ENV = {
   */
   EXTEND_PROTOTYPES: {
     Array: true,
-    Function: true,
```

Comment:
There are two commits in this PR, and the first makes this unrelated change to remove `Function` from this list and docs. The code deletion for function methods on prototype was done at https://github.com/emberjs/ember.js/pull/19663 but this removal was missed.